### ISSUE TITLE
Doesn't work if atom project is not in root

### ISSUE THREAD

**Issue by zhouhao27** [jacogr/atom-git-control#112]
I have a project like this:

root
     --- doc
     --- test
     --- app

My git root is in root, but my project is in app. The git control doesn't work in this case.

**Comment by joaoponceleao** [comment]
Interesting.
I have the opposite problem. My project root is in `root, the .git root is in `root/src. Git control complains the project is not a git repo.

**Comment by joaoponceleao** [comment]
This seems to be an issue with atom. Had the same problem with atomatigit package.
The project root must be git root for the git functions to work in atom. Would prefer if this wasn't the case.

**Comment by MarcelMue** [comment]
Yes this is partially because of atom functions relying on the git root being the project root.
